src/bot.ts

```diff
@@ -104,17 +130,93 @@ export default (app: Probot) => {
     context.log.info(`Posted comment about new claims: ${result.data.html_url}`);
   });
 
-  // Useful for testing purposes:
-  //
-  //  app.on('pull_request.closed', async (context) => {
-  //    const pullRequestNum = context.payload.pull_request.number;
-  //    const issueComment = context.issue({
-  //      body: `Thanks for CLOSING PR #${pullRequestNum}!`,
-  //    });
-  //
-  //    context.log.info(context.payload.repository.owner);
-  //
-  //    context.log.info('PR HAS BEEN CLOSED');
-  //    await context.octokit.issues.createComment(issueComment);
-  //  });
+  app.on("issue_comment.created", async (context: Context<'issue_comment.created'>) => {
+    const repo = context.payload.repository.name;
+    const owner = context.payload.repository.owner.login;
+    const organization = context.payload.organization?.login ?? "";
+    const sender = context.payload.sender.login;
+    const comment = context.payload.comment.body;
+    const issueNumber = context.payload.issue.number;
+    const issueCreatorId = context.payload.issue.user.id;
+    const htmlURL = context.payload.issue.html_url;
+
+    const isPR = htmlURL?.includes(`/pull/${issueNumber}`);
+
+    // check if comment sender is repo owner
+    if(owner !== sender) {
+      return;
+    }
```

Comment:
if we return when comment sender is NOT the owner, wouldn't that severely limit the scope of who can tag people? 

was this decided explicitly? I don't remember at this point

does it make sense to allow any admin do do this?

Reply:
![image](https://user-images.githubusercontent.com/110413633/190143494-4ab40262-c270-4996-b0aa-196fc82093aa.png)

I guess only github maintainer is able to award GitPOAP

Reply:
did kayla include what a 'maintainer' is anywhere? this was definitely unclear

i would say any user that has `write`, `maintain`, and `admin` access to a repo, OR is the owner should count as a valid person who can issue claims

https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization

Reply:
Would owner be the organization if it's not a user's repo?